### PR TITLE
Bug in javascript/typescript template

### PR DESCRIPTION
**Pull request by rmeena840**
Bug in javascript/typescript template

### CODE REVIEW

File: packages/algob/sample-project/js/package.json [hunk 1]
```diff
@@ -4,9 +4,11 @@
 	"main": "index.js",
 	"license": "MIT",
 	"devDependencies": {
-		"mocha": "^9.2.0"
+		"@types/mocha": "^9.1.0",
+		"mocha": "^9.1.3",
+		"ts-mocha": "^9.0.2"
```
Comment (robert-zaremba) [inline packages/algob/sample-project/js/package.json:9]: ts-mocha and @types/mocha should be definitely only in `ts` project, not `js`

**Review by robert-zaremba — CHANGES_REQUESTED** [review]
please fix he dependencies.

File: packages/algob/sample-project/js/package.json (hunk 1 above)
Comment (rmeena840) [reply packages/algob/sample-project/js/package.json:9]: I have only now kept it to ts project. Removed from js.

File: packages/algob/sample-project/js/package.json [hunk 2]
```diff
@@ -4,9 +4,9 @@
 	"main": "index.js",
 	"license": "MIT",
 	"devDependencies": {
-		"mocha": "^9.2.0"
+		"mocha": "^9.1.3"
```
Comment (rmeena840) [inline packages/algob/sample-project/js/package.json:7]:
```suggestion
		"mocha": "^9.2.0"
```

**Comment by rmeena840** [comment]
Fixed

File: packages/algob/sample-project/ts/.gitignore [hunk 3]
```diff
@@ -0,0 +1,6 @@
+nodenode_modules
```
Comment (vuvth) [inline packages/algob/sample-project/ts/.gitignore:1]:
```suggestion
```

**Pull request merged**